CHANGES.rst

```diff
@@ -1,5 +1,8 @@
 Changelog
 =========
+* next release
```

Comment:
When you're ready to do a release, I think that [semver](http://semver.org/) would dictate that this should be *at least `1.2` if not `2.0`.